lib/sanity/attributable.rb

```diff
@@ -22,33 +29,46 @@ def attributes
         @attributes ||= []
       end
 
-      def default_attributes
-        @defaults ||= {}
+      def aliased_attributes
+        @aliased_attributes ||= {}
       end
 
       private
 
-      def attribute(name, default: nil)
-        attributes << name
-        default_attributes.merge!("#{name}": default)
```

Comment:
By removing this you've broken the defaults. Default values no longer work - you could test this in `bin/console` by defining some arbitrary class, initializing it, and seeing the default value being ignored.

Reply:
My mistake, I totally forgot to do something with the default functionality. I've added a fix for this now.